static/app/components/events/viewHierarchy/index.tsx

```diff
@@ -110,6 +113,14 @@ function ViewHierarchy({viewHierarchy, project}: ViewHierarchyProps) {
     initialSelectedNodeIndex: 0,
   });
 
+  // Scroll to the selected node when it changes
+  useEffect(() => {
+    if (!becauseOfFocus.current) {
+      handleScrollTo(item => item === selectedNode);
+    }
+    becauseOfFocus.current = false;
+  }, [selectedNode, handleScrollTo]);
```

Comment:
I had to add `becauseOfFocus` ref since interactions with the tree itself triggers a `selectedNode` change, firing this `useEffect` and it causes some weirdness in how the tree highlights that node.

This way we only scroll to the node if we interact with the tree, otherwise it's already visible by clicking or comes into view by using the arrow keys.

Reply:
Wait, I can get around this.